examples/CSharp/Tutorial 2.2 - Camera/MathHelper.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Tutorial
+{
+    public static class MathHelper
+    {
+        public static float DegreesToRadians(double degrees)
```

Comment:
See other comment

Reply:
That's what I get for not having centralized these yet, i'll fix them all to the float float standard